WooCommerce/WooCommerceUITests/Mocks/mappings/jetpack-blogs/wc/reviews/products_reviews_all.json

```diff
@@ -22,7 +22,7 @@
                 "status": "hold",
                 "reviewer": "Paytin Lubin",
                 "reviewer_email": "pa@example.com",
-                "review": "<p>Travel in style!<\/p>\n",
+                "review": "Travel in style!",
```

Comment:
Tested and validated that we don't need this for the review to be displayed properly on the screen.